api/idea-card/models/idea-card.settings.json

```diff
@@ -75,10 +58,39 @@
       "type": "string"
     },
     "features": {
-      "type": "string"
+      "type": "string",
+      "required": true
     },
     "experience": {
-      "type": "string"
+      "type": "string",
+      "required": true
+    },
+    "extraInfo": {
```

Comment:
How will this be used?

Reply:
In submission form,  there are four required questions, ideaName, description, experience and features. Frontend will check user input before it send to servers.

Reply:
Is this to store response for the `Anything else you would like to share with us?` question?

Reply:
yes. extraInfo is for that quesition.